Unify expectations writing #1778 (#1833)
act-rules/act-rules.github.io: _rules/video-only-as-media-alternative-fd26cf.md

## Expectation 2

> Each target element is labeled as a video alternative for text on the page. This label is [visible][] and [included in the accessibility tree][].

Reviewer: ```suggestion
Each target element is identified as a video alternative for text on the page by a [visible][] label that is [included in the accessibility tree][].
```

Reviewer: Another option is "Each target element is labeled as a video alternative for text on the page by a [visible][] identifier that is [included in the accessibility tree][].

This second option doesn't use "label" so the note wouldn't be necessary, but not sure if "identifier" is a term that's been used in rules before.

Author: I agree this can be amended, which is the preferred result as I will need to update the other file to match.

Reviewer 2: I think I prefer Kathy's version of this.

> **Note:** The term label does not refer to the `label` element.

Reviewer 3: ```suggestion
```
